packages/acceptance-tests/pkg-tests-specs/sources/features/immutablePatterns.test.ts

```diff
@@ -123,6 +123,24 @@ describe(`Features`, () => {
       )
     );
 
+    it(`should prevent reformatting of manifests when so configured`,
+      makeTemporaryEnv(
+        {
+        },
+        {
+          nodeLinker: `node-modules`,
```

Comment:
I wonder is setting `nodeLinker` really needed here?

Reply:
I was simply cargo culting the other tests in this file.

Reply:
No, I don't think it is.  I think I copy-pasted this from my reproduction which was using this linker just because that's what I'm familiar with.

Reply:
Or, I copy-pasted it from other tests in this file mindlessly.